package.json

```diff
@@ -140,7 +140,7 @@
       }
     },
     "test-ci-next": {
-      "command": "npm run version-check && npm run flow:check && bin/config-check.js && better-npm-run jest --color=false --coverage -i",
+      "command": "npm run version-check && npm run flow:check && bin/config-check.js && better-npm-run jest --color=false --coverage -i --silent",
```

Comment:
One of the tests I have updated introduced an odd React warning. I spent several hours trying to resolve the issue, but was unable to figure out how to get rid of this warning. It isn't anything that would happen in production - it is just a result of running the test. Rather than spending even more time on it, I decided to see what would happen if I just hid the warning via `--silent` and it works. This wasn't causing a problem on Node 14, only on 16, for some reason.

Reply:
would this silence other (potentially important) warnings though?